builder/azure/chroot/shared_image_gallery_destination.go

```diff
@@ -21,7 +21,7 @@ type SharedImageGalleryDestination struct {
 
 	TargetRegions         []TargetRegion `mapstructure:"target_regions"`
 	ExcludeFromLatest     bool           `mapstructure:"exclude_from_latest"`
-	ExcludeFromLatestTypo bool           `mapstructure:"exlude_from_latest" undocumented:"true"`
+	ExcludeFromLatestTypo bool           `mapstructure:"exclude_from_latest" undocumented:"true"`
```

Comment:
Hey @lzandman this is a breaking change here, if you notice this field is called "ExcludeFromLatestTypo", and its labeled as undocumented, not really sure why this is here without doing some git spelunking, but can you revert this change?

Reply:
I think I then have to revert all three `shared_image_gallery_destination*` files, right? They seem related.